src/components/UserSignIn/index.tsx

```diff
@@ -37,7 +38,9 @@ const UserSignUp: React.FC = () => {
   return (
     <div className="flex items-center justify-center min-h-screen bg-gray-100">
       <div className="w-full max-w-sm p-6 bg-white rounded-lg shadow-md">
-        <h2 className="text-2xl font-bold text-center text-gray-700">Login</h2>
+        <h2 className="text-2xl font-bold text-center text-gray-700">
+          Register
+        </h2>
         <form className="mt-4" onSubmit={handleSubmit}>
           <div>
             <label className="block text-sm">Email</label>
```

Comment:
qui hai una `<label>` non associata a nessun campo. si usa l'attributo html `for` per associare una label al relativo campo tramite id

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for